osf_tests/test_node.py

```diff
@@ -2465,6 +2467,94 @@ def test_multiple_privacy_changing(self, project):
         assert not project.is_public
 
 
+class TestCheckResourceForSpamPostcommit:
+
+    @pytest.fixture()
+    def user(self):
+        return UserFactory()
+
+    @pytest.fixture()
+    def project(self, user):
+        return ProjectFactory(creator=user)
+
+    @pytest.fixture()
+    def request_headers(self):
+        return {
+            'Remote-Addr': '1.2.3.4',
+            'User-Agent': 'Mozilla/5.0',
+            'Referer': 'https://osf.io'
+        }
+
+    @mock.patch.object(settings, 'SPAM_SERVICES_ENABLED', True)
+    @mock.patch('osf.external.spam.tasks._check_resource_for_domains')
+    def test_check_resource_for_spam_postcommit_with_spammy_domains(self, mock_check_domains, project, user):
```

Comment:
THe ticket instructions to reproduce the bug are:
 > 1. Create project
 > 2. Update wiki with spammy domain
 > 3. Make project public
 
 These are the steps the tests should follow, you are testing behavior that is too specific we know checking for spam domans works, when a project is saved when public, but we need to check a project's wiki content too, not just it's desciption etc.